dashboard/config/overlays/ci-openbsd-pf/etc/rc.conf.local

```diff
@@ -0,0 +1,10 @@
+# Overrides the minimal config in the image created by
+# tools/create-openbsd-vmm-worker.sh to enable PF.
+cron_flags=NO
```

Comment:
As I asked on the issue: don't we hardcode too much here?
We could also put `pf=YES` to sysctl file and then append/replace these entries in /etc/rc.conf.local.

Reply:
Let's keep chatting in #1030, I added some thoughts there.